animation_nodes/nodes/vector/transform_vector.py

```diff
@@ -6,19 +6,27 @@ class TransformVectorNode(bpy.types.Node, AnimationNode):
     bl_label = "Transform Vector"
 
     useVectorList: VectorizedSocket.newProperty()
+    useMatrixList: VectorizedSocket.newProperty()
 
     def create(self):
         self.newInput(VectorizedSocket("Vector", "useVectorList",
             ("Vector", "vector"),
             ("Vectors", "vectors", dict(dataIsModified = True))))
```

Comment:
`dict(dataIsModified = True)` is no longer needed.

Reply:
Done